doc/source/change_proposal_process.rst

```diff
@@ -70,6 +55,11 @@ These steps are necessary to initialize a project repo enabling you to contribut
    be replaced by something else, for example “umn” or
    “api”. Experienced users can configure this in “tox.ini”.
 
+
+Editing the documents
+=====================
+
```

Comment:
Maybe here adding some info that starting from this point steps describe regular editing process (for any other change)

Reply:
yes I wrote a short intrudiction, but I didn´t quite got my head around your request